src/plugins/action_grant_permissions/src/plugin.json

```diff
@@ -4,8 +4,9 @@
   "name": "Grant Permissions",
   "file": "grantPermissions",
   "function_name": "grantPermissions",
-  "description": "Grants permission to user, creates permission join from user to permission name",
-  "required_parameters": ["username", "permissionNames"],
+  "description": "Grants permission to user/org, creates permission join from user/org to permission name",
```

Comment:
would be good to say that if orgName is missing, it will grant permission to user and org combination, otherwise just to user.  Also orgName is not required parameter ? Do we need an optional parameter field ?

Reply:
> would be good to say that if orgName is missing, it will grant permission to user and org combination, otherwise just to user

Okay, done 68d56332

> Also orgName is not required parameter ? Do we need an optional parameter field ?

Ah, you've spotted this! Yeah, it's been bugging me for a bit now, the parameters aren't actually "required", they're just "available". Fortunately, this list is not actually used at all -- it's more for informational purposes, the triggerProcess evaluates whatever parameters that come in from the templateAction. 

This field is stored to the database thought (in action_plugin), with the idea that they would be used in the UI for the TemplateBuilder.

**3 options**:
1. Leave as is for now, worry about it later
2. Change the field name to "available_parameters" (or something similar)
3. Split it into two fields -- "required_parameters" and "optional_parameters"

Options 2 or 3 would require a little change to the schema and the "registerPlugins" method.

Reply:
Will merge this now, but let me know and I'll make an issue to do the above.